src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -730,3 +731,27 @@ export function updateBreadcrumb(breadcrumb) {
     dispatch({ type: UPDATE_BREADCRUMB, breadcrumb });
   };
 }
+/**
+ * Function to retrieve provider information from the provider
+ * endpoint when using the v2 api.
+ *
+ * @param {*} appointments
+ */
+export function getProviderInfoV2(appointments) {
+  // Provider information included with v2 provider api call.
+  return async (dispatch, getState) => {
+    const featureVAOSServiceCCAppointments = selectFeatureVAOSServiceCCAppointments(
+      getState(),
+    );
+    if (featureVAOSServiceCCAppointments) {
```

Comment:
```suggestion
    if (featureVAOSServiceCCAppointments && appointments.practitioners.length > 0) {
```
Add a check to prevent parsing an empty `practitioners` array. Empty array happens when user does not select a provider